app/src/cc/arduino/autocomplete/BaseCCompletionProvider.java

```diff
@@ -0,0 +1,17 @@
+package cc.arduino.autocomplete;
+
+import org.fife.ui.autocomplete.DefaultCompletionProvider;
+
+/**
+ * Base completion provider for C/C++.
+ * @author Ricardo JL Rufino (ricardo@criativasoft.com.br)
```

Comment:
May you add a GNU GPL license on top of all the files and move the credits there?
You can copy the header from any of the other files.